docs/src/main/asciidoc/spring-cloud-netflix.adoc

```diff
@@ -2619,6 +2619,9 @@ certain Ribbon properties.  The properties you can use are
 `client.ribbon.OkToRetryOnAllOperations`. See the https://github.com/Netflix/ribbon/wiki/Getting-Started#the-properties-file-sample-clientproperties[Ribbon documentation]
 for a description of what there properties do.
 
+WARNING: Enabling `client.ribbon.OkToRetryOnAllOperations` includes retring POST requests wich can have a huge impact
```

Comment:
Rather than saying it will have a "huge" impact on server resources, I think it might be more useful to explain why server resources would be effected.